types/react/index.d.ts

```diff
@@ -269,8 +295,25 @@ declare namespace React {
     function isValidElement<P>(object: {} | null | undefined): object is ReactElement<P>;
 
     const Children: ReactChildren;
-    const Fragment: ComponentType;
-    const StrictMode: ComponentType;
+    const Fragment: ExoticComponent<{ children?: ReactNode }>;
+    const StrictMode: ExoticComponent<{ children?: ReactNode }>;
+    /**
+     * This feature is not yet available for server-side rendering.
+     * Suspense support will be added in a later release.
+     */
+    const Suspense: ExoticComponent<{
+        children?: ReactNode
+
+        /** A fallback react tree to show when a Suspense child (like React.lazy) suspends */
+        fallback: NonNullable<ReactNode>|null
```

Comment:
React _will_ throw at runtime if the Suspense suspends and the fallback is undefined.